cmd/zt_sync_local_blob_test.go

```diff
@@ -468,3 +469,53 @@ func TestDryrunSyncLocaltoBlob(t *testing.T) {
 		a.True(testDryrunStatements(blobsToDelete, msg))
 	})
 }
+
+// TestSyncGenerateWarningForLocalToFSUpload
+// @brief This tests performs sync operation from local to file share or vice versa.
+// @validation This test verifies if the warning message is displayed on console if the user tries to perform sync operation from local to file share.
+func TestSyncGenerateWarningForLocalToFSUpload(t *testing.T) {
```

Comment:
Can you add this to the new e2e test framework instead of as a unit test?

Reply:
Added to e2e test framework,